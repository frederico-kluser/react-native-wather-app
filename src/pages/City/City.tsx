import React from 'react';
import Container from './City.styled';

const City = ({children}: any) => <Container>{children}</Container>;

export default City;
