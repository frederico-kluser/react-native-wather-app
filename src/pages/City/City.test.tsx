import React from 'react';
import renderer from 'react-test-renderer';
import City from './City';

test('renders correctly', () => {
  const tree = renderer.create(<City />).toJSON();
  expect(tree).toMatchSnapshot();
});
