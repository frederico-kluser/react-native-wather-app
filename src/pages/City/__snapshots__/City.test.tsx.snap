// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 16,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 16,
      },
    ]
  }
/>
`;
